packages/react-components/src/organisms/TeamCreateOutputHeader.tsx

```diff
@@ -21,18 +21,97 @@ const visualHeaderStyles = css({
   maxWidth: `${720 / perRem}em`,
 });
 
+const headerCopy = ({ outputType }: { outputType: ResearchOutput['type'] }) => {
```

Comment:
Can we do this in a way

const outputTypeMap = {
  'Protocol': 'Share a protocol'
...
}

return outputTypeMap[outputType] ?? 'Share a resource';

Reply:
I think its been done because the URL can have anything technically and there needs to be a default case.

Reply:
Might make sense to have it case insensitive though

Reply:
@niezgoda I think both approaches are okay and there isn't a clear winner. So I'd opt with going what we already have.
@kmaid In this case the outputType is typed, so it can only contain specific values. The url parameter is also typed, so the only way for it to be wrong is if the user manually modifies it. I think we are fine without checking the url parameter for case

Reply:
@kmaid What part do you mean? It is already using `ResearchOutput['type']`. Do you mean the entire parameter type `{ outputType: ResearchOutput['type'] }`? Where would you put shared types like that?

Reply:
Ah, nevermind. Not sure why you are taking an object

Reply:
I usually default to objects because it makes refactoring easier when adding parameters. I can remove it if you want 👍